routing/routing_session.hpp

```diff
@@ -105,6 +106,15 @@ class RoutingSession
   inline void SetState(State state) { m_state = state; }
 
   Route const & GetRoute() const { return m_route; }
+  /// \returns true if any altitude information along |m_route| is available and
+  /// false otherwise.
+  bool HasRouteAltitude() const;
+  /// \brief copies route altitude information to |routeAltitudes| if any is available and
+  /// returns true. If no route altitude information is available returns false.
```

Comment:
Am I right that even if some part of path has altitudes and some part has not, the function returns true? What's the purpose to return false? As I remember, when some part of map is not covered by srtm, we use minimum mwm height for this part.